compiler/include/passes.h

```diff
@@ -141,6 +141,10 @@ bool checkIteratorFromForExpr(Expr* ref, Symbol* shape);
 // LoopExpr.cpp
 bool isOuterVarLoop(Symbol* sym, Expr* enclosingExpr);
 
+// errorHandling.cpp
+Symbol*   findErrorVarForHandlerLabel(LabelSymbol* handlerLabel);
+GotoStmt* gotoForallErrorHandler(ForallStmt* fs);
```

Comment:
Need comments about what gotoForallErrorHandler function does - it is named like it doesn't have side effects but it does add stuff to the AST. What does it add?